sdk/metrics/src/test/java/io/opentelemetry/sdk/metrics/CardinalityTest.java

```diff
@@ -11,13 +11,17 @@
 import io.opentelemetry.api.metrics.LongCounter;
 import io.opentelemetry.api.metrics.Meter;
 import io.opentelemetry.api.metrics.ObservableLongMeasurement;
+import io.opentelemetry.internal.testing.slf4j.SuppressLogger;
 import io.opentelemetry.sdk.testing.exporter.InMemoryMetricReader;
 import java.time.Duration;
 import java.util.concurrent.atomic.AtomicLong;
 import java.util.function.Consumer;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
+@SuppressLogger(
+    loggerName = "io.opentelemetry.sdk.metrics.internal.state.AsynchronousMetricStorage")
+@SuppressLogger(loggerName = "io.opentelemetry.sdk.metrics.internal.state.DeltaMetricStorage")
```

Comment:
I also extended the `@SuppressLoggert` to accept the name of logger to suppress, for cases where the class is package private.